index.js

```diff
@@ -0,0 +1,22 @@
+function openNav() {
+  document.getElementById("mySidepanel").style.width = "200px";
+}
+
+function closeNav() {
+  document.getElementById("mySidepanel").style.width = "0";
+}
+
+let task = [];
+
+console.log(task);
+
+let submit = document.getElementById("submit-button");
+
+submit.addEventListener("click", () => {
+  task.push({
+    id: task.length + 1,
+    title: document.getElementById("task-title").value,
```

Comment:
`document.getElementById("task-title")` this can be a separate variable out of this scope like `submit`.